08_traits_and_reading_bytes.md

```diff
@@ -124,19 +165,25 @@ fn main() {
 }
 ```
 
-And voila, this will print `Version: 1` as expected! Great job so far! 
+And voila, this will print `Version: 1` as expected!
+Great job so far! 
 
-How do we grab the modified `bytes_slice` and continue decoding the transaction? What we probably want to do is pass in the `bytes_slice` into this function as an argument and continue using it in the `main` function. We'll talk more about that and associated Rust concepts of references and borrowing in the next section.
+How do we grab the modified `bytes_slice` and continue decoding the transaction?
+What we probably want to do is pass in the `bytes_slice` into this function as an argument and continue using it in the `main` function.
+We'll talk more about that and associated Rust concepts of references and borrowing in the next section.
 
 ### Quiz
-1. *Take another look at the `Read` trait and the implementation of the `Read` trait for a slice in the documentation. What are the required and provided methods for the trait? What provided methods are being overwritten by the slice?*
+1. *Take another look at the `Read` trait and the implementation of the `Read` trait for a slice in the documentation.
```

Comment:
Out of curiosity, are you taking a look at the `quiz_solutions` folder in this repo as well? It has answers and explanations in there.

Reply:
Of course not... 😅